cy/mv.py

```
@@ -0,0 +1,180 @@
+
+    # Return the specified part or the original list
+    for part_name, part_pages in parts_dict.items():
+        if part_name in sys.argv:
+            printe.output(f"<<yellow>> part: {part_name}: {len(part_pages):,}")
+            return part_pages
+
+    return pages
+
+
+def main2(*args):
+    generator = gent.get_gent(listonly=True, *args)
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)